tests/plugins/simple_text_ckeditor/test_cms_plugins.py

```diff
@@ -7,7 +7,6 @@
 from cms.api import add_plugin
 from cms.models import Placeholder
 from cms.plugin_rendering import ContentRenderer
-
```

Comment:
I think it was nice to separate richie from the other third party dependencies... This change is probably because you now install Richie as the other third parties so isort can't differentiate anymoroe.
See FIRSTPARTY vs THIRDPARTY [here](https://github.com/openfun/richie/blob/master/setup.cfg#L125).

I think you now need to do the same for richie as we do for django:
```
known_django=django
known_richie=richie
sections=FUTURE,STDLIB,DJANGO,THIRDPARTY,RICHIE,LOCALFOLDER
```

Reply:
Well spotted! I'll change this.